lib/beautiful/xresources.lua

```diff
@@ -145,6 +145,7 @@ end
 -- @tparam[opt] integer|screen s The screen.
 -- @treturn integer Resulting size (rounded to integer).
 function xresources.apply_dpi(size, s)
+    if not size then return size end
```

Comment:
Why is this needed? Which code would pass a `nil` (or `false`) here?

Reply:
in my `mixed-dpi` branch I remove a number of uses of the global `apply_dpi`, shifting the computation of the dpi scaling at use time rather than define time. There are a few cases in which the value is optional with a number of fallbacks that can all be nil (`local somevar = args.stuff or default.stuff or blah.stuff`). Rather than leaving that and then adding a new `somevar = somevar and dpi(somevar, s) or nil` line I decided to just wrap everything in a dpi(..., s), which then needs to pass through nil values.

Reply:
Okay, but I guess then this behaviour should be added to the documentation of this function.